Formula/t/thors-mongo.rb

```diff
@@ -28,13 +28,18 @@ class ThorsMongo < Formula
   def install
     ENV["COV"] = "gcov"
 
+    homebrewmacclt = "99999"
```

Comment:
Hum I don't think that's acceptable. Why does it need this information, and what should be given when it is built outside homebrew, or when the CLT is not present?

Reply:
The problem is that in C++17 there is the command `[std::to_chars()](https://en.cppreference.com/w/cpp/utility/to_chars)`

This works on the modern compilers.
But the older Mac (clang) compilers (with command line tools below 15.3) this function is not implemented.

In the code I have added an alternative to cope with situations when the compile time tools do not support this function.

By passing the flag `--with-homebrewCLT=#{homebrewmacclt}` I validate that the version is above 15.3. 

https://github.com/Loki-Astari/ThorsSerializer/blob/master/configure.ac#L21-L38

If it is not then I set the macro `HOMEBREW_OLD_VERSION_OF_MAC`. This is checked in the code here:

See: https://github.com/Loki-Astari/ThorsSerializer/blob/master/src/Serialize/JsonPrinter.cpp#L283-L289

Reply:
Note: The flag `--with-homebrewCLT=` only has an affect if the value is below 153 (version 15.3). Otherwise the code is unchanged. This is only useful for older version of the Apple command line tools.

Reply:
Update the configure script to try and auto detect the availability of `std::to_chars()`